src/transport/concentration_model.hh

```diff
@@ -133,6 +133,46 @@ public:
         { return substances_; }
 
 
+        /// @name Set of methods returning vectors of field names using in different assemblations.
+        // @{
+
+        /// Returns vector of field names of Mass assembly.
+        inline std::vector<string> mass_assembly_subset() const {
```

Comment:
assembly subsets should be independent of the model since the dependency is handled automatically. We only need to mention fields used directly in assembly methods.